kratos/includes/model_part.h

```diff
@@ -797,7 +797,7 @@ class KRATOS_API(KRATOS_CORE) ModelPart : public DataValueContainer, public Flag
     void AddProperties(PropertiesType::Pointer pNewProperties, IndexType ThisIndex = 0);
 
     /** Returns the Properties corresponding to it's identifier is defined */
-    bool HasProperties(IndexType PropertiesId, IndexType ThisIndex = 0) const
+    bool HasProperties(IndexType PropertiesId, IndexType ThisIndex = 0)
```

Comment:
this does not compile?

Reply:
No, that why I revrted

Reply:
It should compile....

Reply:
Apparently the python export is the problem